invokeai/frontend/web/src/features/gallery/components/ImageGrid/GalleryImage.tsx

```diff
@@ -80,15 +91,17 @@ const GalleryImage = (props: HoverableImageProps) => {
           draggableData={draggableData}
           isSelected={isSelected}
           minSize={0}
-          onClickReset={handleDelete}
+          onClickReset={togglePinnedState}
           imageSx={{ w: 'full', h: 'full' }}
           isDropDisabled={true}
           isUploadDisabled={true}
           thumbnail={true}
           withHoverOverlay
-          resetIcon={<FaTrash />}
-          resetTooltip="Delete image"
-          withResetIcon={shouldShowDeleteButton} // removed bc it's too easy to accidentally delete images
+          resetIcon={
```

Comment:
@psychedelicious any thoughts on this interaction? Would need to remove the option to have delete icon here